pkg/resources/secret_with_oauth_client_credentials_acceptance_test.go

```diff
@@ -297,3 +297,62 @@ func TestAcc_SecretWithClientCredentials_EmptyScopesList(t *testing.T) {
 		},
 	})
 }
+
+func TestAcc_SecretWithClientCredentials_ExternalSecretTypeChange(t *testing.T) {
```

Comment:
What would happen if we would change this object into `oauth_authorization_code_grant`, since they have the same secret type?